tests/AppForeach.Framework.Tests/Entity Specification/PrimitiveFieldSpecificationTests.cs

```diff
@@ -6,7 +6,7 @@ namespace AppForeach.Framework.Tests
 {
     public class PrimitiveFieldSpecificationTests
     {
-        readonly PrimitiveFieldSpecification<int> _specification = new PrimitiveFieldSpecification<int>();
+        readonly PrimitiveFieldSpecification<int> _specification = new PrimitiveFieldSpecification<int>(new FacetBag());
```

Comment:
@viktor-kulkov  what if we stick to an empty constructor here new PrimitiveFieldSpecification<int>(), which in turn instntiates **new FacetBag()**